torch/_torch_docs.py

```diff
@@ -8489,6 +8489,12 @@ def merge_dicts(*dicts):
 
 The inverse of this function is :func:`~torch.ifft`.
 
+.. deprecated:: 1.7.0
+    The function :func:`torch.fft` is deprecated and will be removed in a
+    PyTorch 1.8. Use the new :mod:`torch.fft` module functions instead, by
+    importing :mod:`torch.fft` and calling :func:`torch.fft.fft` or
```

Comment:
I'll note that `torch.fft` unfortunately links to the function instead of the module. I was hoping `:mod:` would disambiguate, but I guess it's unlikely to work properly until the function is removed.

Reply:
Darn. I guess we can just write the url by hand for now? It's very unlikely to change.

Reply:
Used a reference so I didn't have to hard-code the URL.

Reply:
Smart solution.